pixi.toml

```diff
@@ -5,44 +5,40 @@ platforms = ["linux-64", "osx-arm64", "osx-64", "win-64"]
 
 [tasks]
 postinstall = "pip install --no-build-isolation --no-deps --disable-pip-version-check -e ."
-coverage = "pytest tests --cov=metalearners --cov-report=xml --cov-report term-missing --color=yes"
-# Note that this task will fail on systems without bash.
-nb-convert = "nb-convert.sh"
-nb-mypy = "mypy docs/examples/*.py"
 
 [dependencies]
```

Comment:
The `depedencies` section is now equal to the [conda-forge recipe](https://github.com/conda-forge/metalearners-feedstock/blob/main/recipe/meta.yaml#L24) except for:
- An extra upper bound on scikit
- The absence of `typing-extensions`.

My hunch would be that `typing-extensions` doesn't actually belong in the recipe's run requirements but I'm not sure.